core/domain/suggestion_services_test.py

```diff
@@ -1852,6 +1852,30 @@ def test_get_reviewable_translation_suggestions_with_language_filter(
         expected_language_code_list: List[str] = []
         self.assertEqual(actual_language_code_list, expected_language_code_list)
 
+    def test_gets_target_ids_of_reviewable_translation_suggestions_for_user(
```

Comment:
Maybe add another test to check that suggestions with language codes not in the allowed set are not considered.

If you like, it's OK (for that test) to mock the storage-layer function and check that it gets called with the right arguments.

Reply:
Done. Although I didn't use mocks in my test.